NoteFind/View Controllers/FeedViewController.m

```diff
@@ -51,4 +54,14 @@ - (IBAction)newPost:(id)sender {
     [self performSegueWithIdentifier:@"newPost" sender:nil];
 }
 
+- (IBAction)didTapLogout:(id)sender {
+    SceneDelegate *sceneDelegate = (SceneDelegate *)self.view.window.windowScene.delegate;
+    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
+        // PFUser.current() will now be nil
+    }];
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
```

Comment:
If logout fails, it might be pre-mature to transition them to the login screen

Reply:
resolved